diff --git a/debian/control b/debian/control
--- a/debian/control
+++ b/debian/control
@@ -4,7 +4,7 @@ Uploaders: Julien Lamy <lamy@unistra.fr>
 Section: science
 Testsuite: autopkgtest-pkg-python
 Priority: optional
-Build-Depends: debhelper-compat (= 12),
+Build-Depends: debhelper (>= 11),
                d-shlibs (>= 0.81),
                dh-python,
                cmake,
@@ -20,6 +20,7 @@
                libboost-filesystem-dev,
                libboost-log-dev,
                libboost-python-dev,
+               libboost-regex-dev,
                pybind11-dev,
                python3-pybind11,
                libboost-test-dev,
diff --git a/debian/compat b/debian/compat
--- /dev/null
+++ b/debian/compat
@@ -0,0 +1 @@
+11
diff --git a/debian/patches/series b/debian/patches/series
--- a/debian/patches/series
+++ b/debian/patches/series
@@ -1,3 +1,2 @@
 remove-soname-in-wrappers.patch
 remove_boost_exception.patch
-fix-interpreter-path.patch
