diff --git a/debian/control b/debian/control
--- a/debian/control
+++ b/debian/control
@@ -4,11 +4,11 @@ Uploaders: Julien Lamy <lamy@unistra.fr>
 Section: science
 Testsuite: autopkgtest-pkg-python
 Priority: optional
-Build-Depends: debhelper-compat (= 12),
-               d-shlibs (>= 0.81),
+Build-Depends: debhelper (>= 10),
+               d-shlibs,
                dh-python,
                cmake,
-               g++ (>= 4:7),
+               g++ (>= 4:6),
                pkg-config,
                libjsoncpp-dev,
                libdcmtk-dev | libdcmtk2-dev,
diff --git a/debian/rules b/debian/rules
--- a/debian/rules
+++ b/debian/rules
@@ -100,6 +100,9 @@ --devunversioned \
 		--commit --multiarch --exclude-a --exclude-la \
 		--movedev debian/tmp/usr/include/* usr/include \
 		--override s/libboost_log.*-dev/libboost-log-dev/ \
+		--override s/libjsoncpp.-dev/libjsoncpp-dev/ \
+		--override 's/lib\(CharLS\|cmr\|dcm\|i2d\|ijg\|of\).*-dev/libdcmtk-dev/' \
+		--override s/libicuuc.*-dev/libicu-dev/ \
 		debian/tmp/usr/lib/*.so
 
 override_dh_clean:
diff --git a/debian/compat b/debian/compat
--- /dev/null
+++ b/debian/compat
@@ -0,0 +1 @@
+10
diff --git a/debian/libodil-doc.doc-base b/debian/libodil-doc.doc-base
--- a/debian/libodil-doc.doc-base
+++ a/debian/libodil-doc.doc-base
@@ -5,5 +5,5 @@
 Section: Science/Medicine
 
 Format: HTML
-Index: /usr/share/doc/libodil-dev/doxygen/html/index.html
-Files: /usr/share/doc/libodil-dev/doxygen/html/*.html
+Index: /usr/share/doc/libodil-doc/doxygen/html/index.html
+Files: /usr/share/doc/libodil-doc/doxygen/html/*.html
